AMTGBot/Bot.cs

```diff
@@ -11,12 +14,16 @@ public class Bot
         public ITelegramBotClient BotClient { get; }
         public BotConfig BotConfig { get; }
         public ILatexRenderer LatexRenderer { get; }
+        public Logger Logger { get; private set; }
```

Comment:
I'm lost, are we into DI or not? What I'm saying if we need 100% of SOLID, we might want to make them all interfaces.

Reply:
That's true. Gonna make it ILogger.